packages/nextjs/utils/scaffold-eth/networks.ts

```diff
@@ -0,0 +1,24 @@
+import { Network } from "@ethersproject/networks";
+
+export function getBlockExplorerTxLink(network: Network, txnHash: string) {
+  const { name, chainId } = network;
+
+  if (chainId === 31337 || chainId === 1337) {
+    // If its localChain then return empty sting
+    return "";
+  }
+
+  let etherscanNetwork = "";
+  if (name && chainId && chainId > 1) {
+    etherscanNetwork = name + ".";
+  }
+
+  let etherscanBaseTxUrl = "https://" + etherscanNetwork + "etherscan.io/tx/";
+  if (chainId && chainId === 100) {
```

Comment:
This can just be `if (chainId === 100)`. Same in line 12, we can remove the chainId condition.